.editorconfig

```diff
@@ -0,0 +1,16 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
Isn´t this colliding with eslint's "indent" rule set to "tab"?

Reply:
agreed, fixed